src/index.ts

```diff
@@ -1,3 +1,3 @@
-export async function hello() {
-  return Promise.resolve("world");
+export async function hello(): Promise<string> {
+  return await Promise.resolve("world");
```

Comment:
why the await here? just return the promise...

Reply:
The typescript linter was complaining that we had an async function without an await inside of it and when I remove the async/await it was complaining that Functions that return promises must be async

Reply:
Looking at this again, we should just return the string here right?

Reply:
actually, yes, since the async keyword is used, we could. but actually, the linter is telling us something useful: you don't need the async keyword here per se, just return Promise.resolve (and specify a promise return type), or just return the string, and leave the async keyword... not sure off the top of my head which one the linter prefers

Reply:
The default setting didn't like that we were using the promise api without `then().catch()`.  It was failing with the `@typescript-eslint/no-floating-promises` rule.
My intuition is to stick to the async/await pattern as much as we can, but this is kind of a weird spot since it's just a filler hello world situation.

Reply:
You latest push is perfect.